sharktank/sharktank/layers/configs/llm_configs.py

```diff
@@ -179,3 +179,72 @@ class LlamaModelConfig:
     # be the difference of many gigabytes of static data being embedded in
     # the program and not.
     static_tables: bool = True
+
+
+@dataclass
+class T5Config:
+    return_dict: bool = True
```

Comment:
This is not being used elsewhere. Also, possible to stay consistent with the format `LlamaHParams` returns?

Reply:
It is possible to refactor out something the is common to Llama and T5. T5 does not have rotary embedding layer and and has `relative_buckets_count`. In general we could make dataclass hierarchy that reflects the GGUF properties. On the other hand there is already the properties dictionary if you want general. These config classes are tailored towards the model. There is some potential for code reuse. I would rather do that later.